lcls-twincat-optics/lcls_twincat_optics_plc/Version/Global_Version.TcGVL

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.6">
   <GVL Name="Global_Version" Id="{f98aab9b-240c-4c69-8c0a-e5732bdbece6}">
     <Declaration><![CDATA[{attribute 'TcGenerated'}
 {attribute 'no-analysis'}
 {attribute 'linkalways'}
 // This function has been automatically generated from the project information.
 VAR_GLOBAL CONSTANT
-	{attribute 'const_non_replaced'}
-	stLibVersion_lcls_twincat_optics : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, sVersion := '0.0.0');
+    {attribute 'const_non_replaced'}
+    stLibVersion_lcls_twincat_optics : ST_LibVersion := (iMajor := 0, iMinor := 6, iBuild := 1, iRevision := 0, nFlags := 0, sVersion := '0.6.1');
```

Comment:
Did you update and install this manually instead of using tc-release?

Reply:
I did not run `tc-release`. Should that happen after merge? I was going to make the release manually on github.

Reply:
`tc-release` requires that the version in the project start at `0.0.0`
the purpose is so that no dev build is marked with a specific tag

Reply:
The proper procedure then was not to "save and install the library" as 0.6.1. on the PLC, but to build the project with 0.0.0 and use `tc-release`? tc-release will do the save the lib version which will make it available to projects?

Reply:
1. library project remains at 0.0.0, keeping it obvious that it is a dev version
2. library project PR is merged
3. `tc-release` creates a tag that includes the project version
4. check out the tag on plcprog-console
5. build as library and install
6. download the library build
7. create the github release including the library build as a file

For provisioning a clear fix I think it's fine to set the version number to match what the release will be as you've done here but for merging back to github we should follow the normal procedure